student-info-system/sis/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from phone_field import PhoneField
+
+
+class Semester(models.Model):
+    name = models.CharField('Semester Name', max_length=20, default='xxx')
+    date_registration_opens = models.DateField('Registration Opens')
```

Comment:
Are we sure we want to set this explicitly like this as it could be different for different people. (Often colleges will allow upper classmen to start registration early)